syntax = "proto3";
message ArticleList {
    repeated ArticleDetail articles = 1;
}
message ArticleDetail {
    string uri = 1;
    string title = 2;
    string source = 3;
    string imageUrl = 4;
    string time = 5;
    string category = 6;
    string body = 7;
}
